### DISCUSSION TITLE
You have to be exactly on top of a small icon to get it to fire, there's no wiggle room!

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13955] [Feature Requests]
I think this is an 'issue', but perhaps it's going to be moved here! There are small icons that you have to place the cursor on top of to be able to get the event to fire and there isn't much padding around it in case you slightly miss it. I think the padding should be made a bit bigger so that you can 'miss' it a bit to get it to fire. Kind of like the whole row linking to a page instead of just the text for web pages.

https://user-images.githubusercontent.com/95102939/174276292-19df2f43-02d7-412d-910c-d0d7fe25cd3b.mp4

First I click exactly on the folding icon, then I click just below it only for it to load the States collection rather than what's closer, and folding the grouping.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
